website/docs/cache.md

```diff
@@ -61,17 +61,27 @@ cache_client_factory.get() # returns None. Multiple client registered.
 ```python
 class Cache(object):
     ...
-    def __init__(self, key: str = None, expire: timedelta = None, invalidate: bool = False, cache_client_name: str = None):
+    def __init__(
+        self,
+        key: str = None,
+        max_key_length: int = 250,
+        expire: Union[timedelta, str] = None,
+        invalidate: bool = False,
+        handler: str = None,
+        cache_empty_result: bool = True,
+    ):
 ```
 * `key` The key used to store the value. possible values are:
    
     `None`: the decorated  `function.__qualname__` is used.     
     
     `some_key{param_1}{param_2}`: will build a key with the `param_1` and `param_2` values.     
     when a parameter is optional and empty `_` is used. 
+* `max_key_length` default `250`, the maximum length of key string to be accepted by decorator.
 * `expire` Period of time when the value is expired.
 * `invalidate` Remove the value from the cache using the key.
-* `cache_client_name` The name to pay to get the correct `CacheHandler`.
+* `handler` name of the handler to be used for handling the cache, by default `Core-Lib`'s default `CacheHandler` is used.
+* `cache_empty_result` type `bool`, default `True`, is responsible for caching the empty values like `{}`, `[]`, `()`, `""` and `set()`.

```

Comment:
`cache_empty_result` type `bool`, default `True`. When `True` will cache empty values as `{}`, `[]`, `()`, `""` and `set()`